385/Assets/Scripts/GroundTimed.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GroundTimed : MonoBehaviour {
+    public float DisappearTime = 3f;
```

Comment:
I would add a `[Range]` attribute here, so we can easily fine tune this value in the editor. A reasonable range would be 0-10 seconds. I would also add comments that explain that this is in whole seconds, and not in milliseconds.